Breaking Config options into files (#2038)
tinymce/tinymce-docs: _includes/configuration/media_poster.md

### `media_poster`

> This options allows you disable the `Poster` input field in the media dialog.

Reviewer: This has some grammar issues and isn't even a proper sentence. I think it's missing `to`?

Author: ```suggestion
To remove the `Poster` input field in the media dialog, set this option to `false`.
```
